base-kustomize/mariadb-operator/base/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources: []
```

Comment:
I think should should be:
```
resources:
  - all.yaml
```

to catch the output of kustomize.sh?

I could be wrong. I didn't understand that machinery super well.

Reply:
That's generally correct but in this case because we don't have any kustomize base or overlay, I created this only as a placeholder for the future changes and that's why it seems kind of empty.